istantestd.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+#
+NPM=$(which npm)
+ROOTPATH=$(pwd)
+$NPM install
+echo "Cloning test.."
+git clone -b authentication https://github.com/OADA/oada-compliance.git 
+cd oada-compliance/
+$NPM install
+
+cd $ROOTPATH
+echo "Starting instrumented server.."
+export PORT=8443
+istanbul cover index.js & 
```

Comment:
You can also give the server your own config.js by calling it like:

`istanbul cover index.js -- path_to_your_config.js`

the magic (pretty fragile) is in _config 

Also, you want the --include-all-sources flag. Maybe we should use the .istanbul.yml file?